test/core/integration/concrete/lockup/burn/burn.t.sol

```diff
@@ -2,177 +2,142 @@
 pragma solidity >=0.8.22 <0.9.0;
 
 import { IERC721Errors } from "@openzeppelin/contracts/interfaces/draft-IERC6093.sol";
+import { IERC4906 } from "@openzeppelin/contracts/interfaces/IERC4906.sol";
 import { ISablierLockup } from "src/core/interfaces/ISablierLockup.sol";
 import { Errors } from "src/core/libraries/Errors.sol";
 import { Integration_Test } from "./../../../Integration.t.sol";
-import { Lockup_Integration_Shared_Test } from "./../../../shared/lockup/Lockup.t.sol";
-
-abstract contract Burn_Integration_Concrete_Test is Integration_Test, Lockup_Integration_Shared_Test {
-    uint256 internal streamId;
-    uint256 internal notTransferableStreamId;
-
-    function setUp() public virtual override(Integration_Test, Lockup_Integration_Shared_Test) {
-        streamId = createDefaultStream();
-        notTransferableStreamId = createDefaultStreamNotTransferable();
-
-        // Make the Recipient (owner of the NFT) the caller in this test suite.
```

Comment:
for the first tests (`test_RevertGiven_Null`, `test_RevertGiven_PENDINGStatus` `test_RevertGiven_STREAMINGStatus` `test_RevertGiven_SETTLEDStatus` `test_RevertGiven_CANCELEDStatus`) i guess the caller will be the admin, but it is irelevant since the caller check is not made before these reverts

Reply:
Fair point and I agree with you.